### ISSUE TITLE
The new update seems to break old saves!

### ISSUE THREAD

**Issue by gsxdsm** [CoenraadS/News-Ticker#3]
Can you see:

http://www.reddit.com/r/CitiesSkylinesModding/comments/2zeiwx/how_i_broke_everyones_savegame_and_fixed_them/

I think this mod is suffering from the same problem..

**Comment by gsxdsm** [comment]
And if the MOD Is uninstalled it breaks old saves - the messages are serialized in the save file:

Simulation error: Unknown type: News_Ticker.NewsTickerMessage, News Ticker, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null
  at ColossalFramework.IO.DataSerializer.ReadSharedType (ColossalFramework.IO.LegacyResolverHandler customResolver) [0x00000] in <filename unknown>:0 
  at ColossalFramework.IO.DataSerializer.ReadObject[MessageBase] (ColossalFramework.IO.LegacyResolverHandler customResolver) [0x00000] in <filename unknown>:0 
  at ColossalFramework.IO.DataSerializer.ReadObjectArray[MessageBase] (ColossalFramework.IO.LegacyResolverHandler customResolver) [0x00000] in <filename unknown>:0 
  at ColossalFramework.IO.DataSerializer.ReadObjectArray[MessageBase] () [0x00000] in <filename unknown>:0 
  at MessageManager+Data.Deserialize (ColossalFramework.IO.DataSerializer s) [0x00000] in <filename unknown>:0 
  at ColossalFramework.IO.DataSerializer.Deserialize[Data] (System.IO.Stream stream, Mode mode, ColossalFramework.IO.LegacyResolverHandler customResolver) [0x00000] in <filename unknown>:0 
  at ColossalFramework.IO.DataSerializer.Deserialize[Data] (System.IO.Stream stream, Mode mode) [0x00000] in <filename unknown>:0 
  at LoadingManager+<LoadSimulationData>c__Iterator55.MoveNext () [0x00000] in <filename unknown>:0 
  at AsyncTask.Execute () [0x00000] in <filename unknown>:0   [Core]

**Comment by gsxdsm** [comment]
See: https://github.com/mabako/reddit-for-city-skylines/issues/2

**Comment by CoenraadS** [comment]
Well actually the old version indeed breaks saved, the new version doesn't.

But it could be that the new version also breaks the old version, I'll take a look at it.

**Comment by gsxdsm** [comment]
Got it thanks! I totally get it now appreciate the response.

**Comment by gsxdsm** [comment]
Yeah the old version doesn't work - still can't load the assembly...the problem is that the gamesave has the reference to the assembly with a space, and since the deserializer can't find it, there is no hope to fix it unless we can modify the save.

Thanks for trying! My bad for not making a save before trying the mod

**Comment by CoenraadS** [comment]
I'm just going to keep this open for now, since other people may find it useful.
